### DISCUSSION TITLE
Instanced Drawing

### DISCUSSION THREAD

**Discussion by koreex** [creativelifeform/three-nebula#184] [General]
I'm interested in the performance of three-nebula. I think instanced drawing would be proper to improve the performance of particle system. But I haven't found out any use of instanced drawing in three-nebula. Do you have any idea? Thanks in advance.

**Comment by rohan-deshpande** [comment]
Hi, `three-nebula` doesn't use instanced drawing yet, I agree it could be used to improve performance, PRs are welcome!

**Reply by koreex** [reply]
Thanks. May I ask what kind of techniques you are using for `GPURenderer`. It calls draw only once.
